front/components/authPage/AuthPage.vue

```diff
@@ -71,6 +76,13 @@ export default {
       default: false
     }
   },
+
+  methods: {
+    goBack() {
+      window.history.back();
```

Comment:
better to use router.
router.go(-1)

Reply:
Okay, Is there any specific reason to use router.go(-1) instead of window.history.back() or is it just for consistency?

Reply:
commenting because i'd also like to learn if there's a difference between these 2 methods. :)